app/styles/ui/_commit-message-avatar.scss

```diff
@@ -88,6 +88,17 @@
     bottom: 13px;
 
     width: 300px;
+
+    @media (max-height: 500px) {
+      bottom: -125px;
```

Comment:
I considered another approach which would be to change the popover from hardcoded positioning to calculated positioning; but, I didn't think the time involved and testing all the popover use cases would be worth it.